src/pages/hpp-profile.astro

```diff
@@ -639,60 +638,62 @@ const internalNavLinks = [
                 </section>
               </article>
 
-              <article class="wrapper hpp-multisystemic__report">
-                <h2
-                  class="hpp-multisystemic__report--title hpp__profile__subhead"
-                >
-                  Selected reported body system signs/symptoms or interventions in children with HPP<sup>9</sup>:
-                </h2>
-
-                <picture>
-                  <source
-                    media="(min-width: 769px)"
-                    srcset="/assets/images/hpp-profile/adolescents-signs-and-symptoms-chart--desktop.png"
-                  />
-                  <img
-                    class="hpp-multisystemic__report--chart"
-                    src="/assets/images/hpp-profile/adolescents-signs-and-symptoms-chart.png"
-                    alt="FPO IMAGE"
-                  />
-                  <figcaption>Percent of patients</figcaption>
-                </picture>
-
-                <p class="hpp-multisystemic__report--conditions">
-                  Multinational, observational, prospective Global Hypophosphatasia Registry study conducted in children (n=121; &lt;18 years) from January 2015 to September 2017.<sup>9</sup>
-                </p>
+              <div class="hpp-multisystemic__report--bg">
+                <article class="wrapper hpp-multisystemic__report">
+                  <h2
+                    class="hpp-multisystemic__report--title hpp__profile__subhead"
+                  >
+                    Selected reported body system signs/symptoms or interventions in children with HPP<sup>9</sup>:
+                  </h2>
 
-                <p
-                  class="hpp-multisystemic__report--conditions hpp-multisystemic__report--conditions-smaller hpp-multisystemic__report--conditions--mt1"
-                >
-                  <sup>a</sup>Excludes patients aged &lt;2 years at enrollment.
-                </p>
+                  <picture>
+                    <source
+                      media="(min-width: 700px)"
+                      srcset="/assets/images/hpp-profile/adolescents-signs-and-symptoms-chart--desktop.png"
+                    />
+                    <img
+                      class="hpp-multisystemic__report--chart"
+                      src="/assets/images/hpp-profile/adolescents-signs-and-symptoms-chart.png"
+                      alt="FPO IMAGE"
+                    />
+                    <figcaption>Percent of patients</figcaption>
```

Comment:
The figcaption is bolder according to figma
![image](https://github.com/HackMort/ALPD/assets/35670846/a9d036f9-bb39-4236-a335-6edc949a6fa0)
Figma:
![image](https://github.com/HackMort/ALPD/assets/35670846/0f7afd46-ecc7-416c-833d-07a6e0132c51)

_Check the others charts too_